1-estrategia/README.md

```diff
@@ -1,23 +1,237 @@
+	* Para solucionar las peticiones de aquellos usuarios que quieran encontrar algún contenido específico, ofreceremos un buscador por palabras o categorías, para lo cual etiquetaremos las palabras por palabras clave. De esta manera facilitaremos que nuestros usuarios puedan encontrar aquello que desean de una manera mucho más eficaz.  
+
+* Utilidad: 
+	* El sitio web permitirá a los usuarios compartir los contenidos que se publiquen a través de aplicaciones de mensajería instantánea. Además, en la parte inferior de cada publicación habrá un espacio destinado a ofrecer la posibilidad a los usuarios de que realicen su feedback a través de comentarios, sugerencias o críticas, por citar algunos ejemplos.  
+
+* Estética: 
+	* Tratándose de un sitio web cuyos fines principales son educar e informar, el diseño estéticamente hablando tendrá un carácter formal. Priorizaremos la correcta visión de todos los elementos que formen nuestro sitio web, en lugar de que visualmente el sitio sea bonito. Optaremos por colores neutros, así como por tipografías clásicas, que sean sencillas de interpretar. 
+
 ### Indicadores de éxito
 
-## Necesidades del usuario
+Creemos que a través de los siguientes indicadores de éxito podremos comprobar si nuestros objetivos se están llevando a cabo de una forma exitosa o no. Además, como cuestionaremos a los usuarios acerca de varios aspectos, podremos identificar los errores y reforzar los aciertos con una posibilidad mayor de éxito. 
+
+* Satisfacción del cliente
+	* Grado de satisfacción con la experiencia navegando en el sitio web (1-5) 
+
+* Grado de recomendación
+	* ¿Recomendarías este sitio web a algún amigo/familiar/conocido (poco recomendable-muy recomendable)? 
+	* NPS: Detractores (0-6), Pasivos (7-8), Promotores (9-10). 
+
+* Usabilidad
+	* ¿Qué tareas has podido completar a través de nuestro sitio web? Elabora una respuesta detallando una o varias tareas que hayas podido realizar
+
+* Dificultades del usuario
+	* ¿Con qué dificultades te has encontrado a la hora de realizar una tarea?
+
+* Descripción general del sitio
+	* ¿Con qué adjetivo/s describirías nuestro sitio web? (Escribir o seleccionar alguno de los que propongamos nosotros)
+
+* Valoración general  
+	* ¿Qué puntuación global le ofrecerías a nuestro sitio web? (1-5)
+
+## NECESIDADES DEL USUARIO
 
 ### Investigación de usuarios
+Como bien hemos comentado, nuestro proyecto web es crear una página web relacionada con la **igualdad de género**. En esta página web se encontrarán noticias, reportajes, datos, videos, etc. Todo el contenido hallado en este sitio web tendrá el siguiente objetivo: informar y educar.
+
+En cuanto a nuestro público objetivo creemos que es bastante amplio, ya que tanto mujeres como hombres mayores de 18 años podrían tener interés en acceder a nuestra página web. Pero antes de crear nuestro producto debemos conocer a nuestros usuarios y sobre todo debemos percibir al máximo sus necesidades. ¿Pero cuáles son esas necesidades? 
```

Comment:
_creemos que es_, _podrían tener interés_... Antes habéis mencionado una horquilla de edad. ¿Dónde queda eso ahora?